geometry_msgs/PoseStamped robot_pose
string robot_name